hibike/devices/ExampleDevice/example_device.cpp

```diff
@@ -38,8 +38,8 @@ void loop() {
   // so exessive delays here will affect hibike.
   //don't change Kumiko.
 
-  hazuki += 0;
-  sapphire += 1;
+  // hazuki += 0;
```

Comment:
I stopped two of the parameters from auto-incrementing so they can be used to test writing instead. `hibike_process.py` will now right two parameters if you run is directly